xapian-letor/Makefile.am

```diff
@@ -113,4 +138,4 @@ noinst_HEADERS +=\
 	common/str.h\
 	common/stringutils.h
 
-DISTCHECK_CONFIGURE_FLAGS = "XAPIAN_CONFIG=$(XAPIAN_CONFIG)"
+DISTCHECK_CONFIGURE_FLAGS = "XAPIANLETOR_CONFIG=$(XAPIANLETOR_CONFIG)"
```

Comment:
I noticed after merging when reviewing #263 (which seems to include a version of the changes here) that this change seems wrong - `xapian-letor` doesn't use `xapianletor-config`, but it ought to use the in-tree xapian-config for the nested build when running `make distcheck`.

Reply:
Can you explain it a bit further as to why it won't use xapianletor-config?.

Reply:
understood.